src/server/main.py

```diff
@@ -45,6 +49,19 @@ def _setup_endpoints(self) -> None:
         self.api.register_blueprint(self.v1)
         self.app.register_blueprint(self.api)
 
+    def _setup_scheduler(self) -> None:
```

Comment:
Could this method be hidden in `DatabaseUpdater` (and start it with `initialize()` method in `__init__()`)? It would keep consistency with other initialized "manager" classes.

Reply:
I would prefer not to move that method outside of the main class. It may be safer to keep the scheduler variable inside of it, so I think it's not worth the risk.

Reply:
To be fair, I don't see the risk of moving the scheduler to `DatabaseUpdater`. It's lifetime will be started at the exact same moment and will remain for the same duration (until program terminates). Is there a specific reason why is it necessary to keep it in the `Server` class?

Reply:
done